frontend/i18n-scripts/export-pos.sh

```diff
@@ -2,22 +2,24 @@
 
 set -exuo pipefail
 
+source ./i18n-scripts/languages.sh
+
 for f in public/locales/en/* ; do
-  yarn i18n-to-po -f $(basename "$f" .json) -l zh-cn
-  yarn i18n-to-po -f $(basename "$f" .json) -l zh-tw
-  yarn i18n-to-po -f $(basename "$f" .json) -l ja
-  yarn i18n-to-po -f $(basename "$f" .json) -l ko
+  for i in "${LANGUAGES[@]}"
+  do
+  yarn i18n-to-po -f "$(basename "$f" .json)" -l "$i"
+  done
 done
 
 cd packages
 for d in */ ; do
-  if [ -d "$d/locales/en" ]
+  if [ -d "$d"/locales/en ]
```

Comment:
I believe this was find as it was. You can quote the entire value, although what you have is OK, too.